invenio_app_ils/documents/utils.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Document utils."""
+
+MAX_AUTHORS = 5
```

Comment:
nit: could it be overridable in the overlay?

Reply:
I thought to create a config for it, but then honestly I thought that it was just too much... do you prefer to have it a config?

Reply:
no I'm fine with that, it's not a blocking issue, and very low impact